docs/AuthenticationMethods.md

```diff
@@ -0,0 +1,107 @@
+#Authentication Methods
```

Comment:
General for the document: for different options, we require different authentication to be configured on the Kusto cluster. We need to clarify what is the appropriate Kusto configuration per option.

Reply:
We currently have the following requirements in KustoSource.md:

- Client application is registered in AAD
- Client application has 'viewer' privileges or above on the target database, or 'admin' privileges on the target table (either one is sufficient)

If we use device (user) authentication - do we need the user to have the appropriate privileges instead of an application?

Reply:
yes